tools/apigee-x-trial-provision/apigee-x-trial-provision.sh

```diff
@@ -17,6 +17,46 @@
 
 set -e
 
+# options
+pps=""
+while(($#)); do
+case "$1" in
+  -p|--project)
+    PROJECT="$2"
+    shift 2;;
+
+  -n|--network)
+    NETWORK="$2"
+    shift 2;;
+
+  -r|--region)
+    REGION="$2"
+    shift 2;;
+
+  -z|--zone)
+    ZONE="$2"
+    shift 2;;
+
+  -x|--ax-region)
+    AX_REGION="$2"
+    shift 2;;
+
+  -c|--certificates)
+    CERTIFICATES="$2"
+    shift 2;;
+    
+  -q|--quiet)
+    QUIET=Y
+    shift;;
+
+  *)
+    pps="$pps $1"
+    shift;;
+esac
+done
+eval set -- "$pps"
```

Comment:
yes, we need pps pretty soon. next refactoring iteration to subsume main util functionality into install operation, then add other operations (wait-for-ready), etc.

then the suggestion is for install to have -c/--component option that would allow skip/chose specific parts.

Reply:
fine. That works